Intersect.Server/Migrations/Game/GameContextModelSnapshot.cs

```diff
@@ -786,7 +788,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.ToTable("Time");
                 });
 
-            modelBuilder.Entity("Intersect.Server.Maps.MapInstance", b =>
+            modelBuilder.Entity("Intersect.Server.Maps.MapController", b =>
```

Comment:
why are you changing this?
pretty sure that MapInstance is what we use now since the latest changes on map's instancing 😕

Reply:
He is not changing that, creating any migration causes this line to change, I can say from experience, every time I create a migration, I manually change this line to MapInstance again, I believe this edit should be done here too, and something should be done about this change that occurs every migration

Reply:
Since this was changed in JCs PR that got merged, this change should be omitted from this PR.